rfcs/20201113-nanodiam-1.0.0.md

```diff
@@ -0,0 +1,122 @@
+# nanoMFG Software Planning Document
+<!-- Replace text below with long title of project:short-name -->
+## Tool for Raman of Nanodiamond: nanodiam
+### Target Release: 1.0.0 : December 31, 2020
+
+## Development Team
+<!-- Complete table for all team members 
+ roles: lead, developer, reviewer
+ status: active, inactive
+-->
+Name | Role | github user | nanohub user | email | status
+---|---|---|---|---|---
+Lili Cai| Lead | | |lilicai@illinois.edu|
+Adrian Manjarrez | Developer | adrianm3 | amanjarrez | adrianm3@illinois.edu | active
+ |  |  |  |  | 
```

Comment:
Here are the details for adding Aagam and I here as developers as per @elifleaf recommendation:
Darren Adams | developer | dadamsncsa | Darren K Adams | dadams@illinois.edu | active
Aagam Shah | developer | AagamShah97 |  aagam2 | aagam2@illinois.edu | active